modules/core/site.js

```diff
@@ -257,7 +257,8 @@ var Hm_Ajax_Request = function() { return {
                 Hm_Ajax.err_condition = false;
                 Hm_Notices.hide(true);
             }
-            if (res.router_user_msgs && !$.isEmptyObject(res.router_user_msgs)) {
+            if (res.router_user_msgs && !$.isEmptyObject(res.router_user_msgs)) {             
+                var hm_msg = Hm_Utils.json_encode(res.router_user_msgs);
```

Comment:
Why do you assign this to a variable and not use it? Can you remove if that's debugging code?
Otherwise, the PR looks good, we can merge after this is fixed.

Reply:
Noted